### PR TITLE
Fix misspelled gprc

### PR DESCRIPTION
**Pull request by y-zeng**

### CODE REVIEW

File: test/core/profiling/mark_timings.stp [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * probe definition.
  *
  * For a statically build binary, that'd be the name of the binary itself.
- * For dinamically built ones, point to the location of the libgprc.so being
+ * For dinamically built ones, point to the location of the libgrpc.so being
```
Comment (sreecha) [inline test/core/profiling/mark_timings.stp]: Can you also correct this:  s/dinamically/dynamically

Comment (y-zeng) [inline test/core/profiling/mark_timings.stp]: Fixed. Thanks for the review!

**Comment by sreecha** [comment]
Lgtm . Please resolve merge conflicts and once all tests pass, you can set the ready to merge flag

**Comment by y-zeng** [comment]
Test failures seem irrelevant.

**Pull request merged**